test/fixtures/basic/pages/nuxt-link/trailing-slash.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+const LinkWithTrailingSlash = defineNuxtLink({
+  trailingSlash: 'append'
+})
+const LinkWithoutTrailingSlash = defineNuxtLink({
+  trailingSlash: 'remove'
+})
+const links = [
+  '/',
+  '/nuxt-link/trailing-slash',
+  '/nuxt-link/trailing-slash/',
+  '/nuxt-link/trailing-slash?test=true&thing=other/thing#thing-other',
+  '/nuxt-link/trailing-slash/?test=true&thing=other/thing#thing-other',
+  { name: 'nuxt-link-trailing-slash' },
+  { query: { 'with-state': 'true' }, state: { foo: 'bar' } },
+  { query: { 'without-state': 'true' } }
+]
+
+const route = useRoute()
+const windowState = computed(() => {
+  console.log(route.fullPath)
```

Comment:
`console.log` intended?

Reply:
Fair play 😆

Reply:
```suggestion
```

Reply:
oops - I forgot why it was there - to make this reactive to the route - otherwise the window history won't be updated when the route changes, IIRC.